test/test18.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+def missing_num(arr):

```

Comment:
It doesn't make sense to write your solution in solutions file, then literally write the same code here. The point of tests is to test your code, so I would import the solutions file. I did a quick google search, perhaps this tutorial would help? https://www.codementor.io/sheena/python-path-virtualenv-import-for-beginners-du107r3o1

By the way, good job. If you pull this off we will officially support python :)